pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+- Related Issue (include '#'):
+
+- Description of changes made:
+
+- Is the feature complete/bug resolved/etc..:
+
+- Any known bugs/strange behavior:
+
+- Is there specific feedback you would like on these changes:
```

Comment:
@caioingber Do we need this file too? It's in the root of the repository

Reply:
@ZakkMan Hmmm not sure why that was included in the PR i moved that file into the template folder

Reply:
I removed it from the pull request

Reply:
Great!